vendor/composer/autoload_static.php

```diff
@@ -61,6 +61,10 @@ class ComposerStaticInit22ddb69348c7ed922c96325249cef3d0
         array (
             'Psr\\Log\\' => 8,
```

Comment:
Actually. Whatever you did, it now looks good. Only are several changes in this script for the Services stuff. :)